plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -47,15 +50,16 @@
   private static final Logger logger = Logger.getLogger(AppEngineStandardFacet.class.getName());
 
   public static final String ID = "com.google.cloud.tools.eclipse.appengine.facets.standard";
-  public static final String VERSION = "1";
 
   public static final IProjectFacet FACET = ProjectFacetsManager.getProjectFacet(ID);
-  public static final IProjectFacetVersion FACET_VERSION = FACET.getVersion(VERSION);
+  public static final IProjectFacetVersion JRE7 = FACET.getVersion("JRE7");
+  // public static final IProjectFacetVersion JRE8 = FACET.getVersion("JRE8");
```

Comment:
Why not having this one and removing `AppEngineStandardFacetChangeListener.APP_ENGINE_STANDARD_JRE8`?